src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -317,10 +323,14 @@ public void checkNewlyActiveArtefacts() {
     }
 
     /**
-     * Scheduled method that checks daily for newly outdated artefacts based on a yesterday or older display to date.
+     * Scheduled method that:
+     *  checks daily for newly outdated artefacts based on a yesterday or older display to date.
+     *  Checks daily for a list of all no match artefacts to send to publication services
      */
     @Scheduled(cron = "${cron.daily-start-of-day}")
     public void deleteExpiredBlobs() {
+        findNoMatchArtefactsForReporting(artefactRepository.findAllNoMatchArtefacts());
```

Comment:
this fees like it should be it its own method, deleting expired blobs and finding all non matching to send for reporting feel like 2 different processes.
theres 2 ways to go about it; 
1) remove this reporting method and put in its own scheduled method
2) rename this method to something like `runDailyTasks` and then extract these as 2 private methods called within the scheduler

Reply:
i mean youve already got it as a separate method, could just add the scheduled annotation onto it

Reply:
Went for option 2 like we discussed on slack